test/autoprefixer.test.js

```diff
@@ -90,6 +90,9 @@ let example = autoprefixer({
 let autofiller = autoprefixer({
   overrideBrowserslist: ['Chrome > 90', 'Firefox >= 82']
 })
+let textDecorator = autoprefixer({
+  overrideBrowserslist: ['Chrome >= 57', 'Firefox >= 36', 'Safari >= 12.1']
+})
```

Comment:
I want a config with browsers that have unprefixed support for :
- `text-decoration-color`
- `text-decoration-line`
- `text-decoration-style`

I want to test the shorthand in isolation

I am not sure I got the naming convention right, so please let me know if I misunderstood.